src/lib/Scrollyteller/PanelObserver.svelte

```diff
@@ -65,40 +65,47 @@
 	}
 
 	// Set up observer for panel position ======================================
+
 	let panelObserver;
 	/**
 	 * Track intersecting panels. We can change the viz back to the last panel
 	 * which we otherwise can't do if there are 2 panels overlapping at once.
 	 */
 	let intersectingPanels = [];
```

Comment:
Should this be inside the function closure? Doesn't seem to be used outside the function.

Reply:
This doesn't apply now you've undone the enfunctionification.